### PR TITLE
Update tutorials to build faster

### LINKED ISSUE #825: Improve some tutorials to speed up the Continuous Documentation
This is a summary of running time of the PyGMT tutorials and examples:

```
computation time summary:
    - ../examples/tutorials/regions.py:                                   11.34 sec   0.0 MB
    - ../examples/tutorials/3d-perspective-image.py:                      10.06 sec   0.0 MB
    - ../examples/tutorials/frames.py:                                     6.56 sec   0.0 MB
    - ../examples/tutorials/lines.py:                                      5.86 sec   0.0 MB
    - ../examples/gallery/grid/grdview_surface.py:                         5.47 sec   0.0 MB
    - ../examples/tutorials/earth-relief.py:                               5.32 sec   0.0 MB
    - ../examples/tutorials/text.py:                                       4.63 sec   0.0 MB
    - ../examples/tutorials/contour-map.py:                                3.88 sec   0.0 MB
    - ../examples/tutorials/coastlines.py:                                 3.07 sec   0.0 MB
    - ../examples/tutorials/plot.py:                                       2.58 sec   0.0 MB
    - ../examples/projections/azim/azim_general_perspective.py:            2.36 sec   0.0 MB
    - ../examples/tutorials/first-figure.py:                               1.71 sec   0.0 MB
    - ../examples/tutorials/configuration.py:                              1.58 sec   0.0 MB
    - ../examples/gallery/coast/borders.py:                                1.45 sec   0.0 MB
    - ../examples/gallery/plot/scatter3d.py:                               1.23 sec   0.0 MB
    - ../examples/gallery/grid/track_sampling.py:                          1.18 sec   0.0 MB
    - ../examples/projections/cyl/cyl_cassini.py:                          0.99 sec   0.0 MB
    - ../examples/projections/azim/azim_equidistant.py:                    0.93 sec   0.0 MB
    - ../examples/gallery/line/line-custom-cpt.py:                         0.91 sec   0.0 MB
    - ../examples/gallery/coast/shorelines.py:                             0.89 sec   0.0 MB
    - ../examples/projections/conic/conic_albers.py:                       0.85 sec   0.0 MB
    - ../examples/gallery/coast/land_and_water.py:                         0.83 sec   0.0 MB
    - ../examples/gallery/plot/image.py:                                   0.82 sec   0.0 MB
    - ../examples/gallery/plot/scatter.py:                                 0.73 sec   0.0 MB
    - ../examples/projections/cyl/cyl_universal_transverse_mercator.py:    0.70 sec   0.0 MB
    - ../examples/gallery/plot/logo.py:                                    0.67 sec   0.0 MB
    - ../examples/projections/azim/azim_gnomonic.py:                       0.67 sec   0.0 MB
    - ../examples/projections/misc/misc_van_der_grinten.py:                0.66 sec   0.0 MB
    - ../examples/projections/nongeo/polar.py:                             0.66 sec   0.0 MB
    - ../examples/projections/azim/azim_orthographic.py:                   0.66 sec   0.0 MB
    - ../examples/projections/cyl/cyl_miller.py:                           0.65 sec   0.0 MB
    - ../examples/projections/azim/azim_general_stereographic.py:          0.65 sec   0.0 MB
    - ../examples/projections/misc/misc_winkel_tripel.py:                  0.64 sec   0.0 MB
    - ../examples/projections/nongeo/cartesian_logarithmic.py:             0.64 sec   0.0 MB
    - ../examples/projections/conic/conic_lambert.py:                      0.63 sec   0.0 MB
    - ../examples/gallery/plot/legend.py:                                  0.62 sec   0.0 MB
    - ../examples/gallery/line/linestyles.py:                              0.61 sec   0.0 MB
    - ../examples/projections/cyl/cyl_equal_area.py:                       0.61 sec   0.0 MB
    - ../examples/projections/cyl/cyl_transverse_mercator.py:              0.58 sec   0.0 MB
    - ../examples/projections/conic/polyconic.py:                          0.58 sec   0.0 MB
    - ../examples/projections/nongeo/cartesian_power.py:                   0.57 sec   0.0 MB
    - ../examples/projections/cyl/cyl_equidistant.py:                      0.57 sec   0.0 MB
    - ../examples/projections/misc/misc_eckertVI.py:                       0.57 sec   0.0 MB
    - ../examples/gallery/plot/colorbar.py:                                0.56 sec   0.0 MB
    - ../examples/gallery/plot/points.py:                                  0.56 sec   0.0 MB
    - ../examples/projections/cyl/cyl_mercator.py:                         0.55 sec   0.0 MB
    - ../examples/projections/misc/misc_eckertIV.py:                       0.55 sec   0.0 MB
    - ../examples/projections/misc/misc_hammer.py:                         0.55 sec   0.0 MB
    - ../examples/projections/azim/azim_lambert.py:                        0.55 sec   0.0 MB
    - ../examples/projections/nongeo/cartesian_linear.py:                  0.55 sec   0.0 MB
    - ../examples/gallery/plot/points-transparency.py:                     0.55 sec   0.0 MB
    - ../examples/projections/cyl/cyl_stereographic.py:                    0.55 sec   0.0 MB
    - ../examples/projections/misc/misc_robinson.py:                       0.55 sec   0.0 MB
    - ../examples/projections/misc/misc_sinusoidal.py:                     0.54 sec   0.0 MB
    - ../examples/projections/misc/misc_mollweide.py:                      0.53 sec   0.0 MB
    - ../examples/projections/conic/conic_equidistant.py:                  0.53 sec   0.0 MB
    - ../examples/gallery/plot/meca.py:                                    0.53 sec   0.0 MB
    - ../examples/gallery/plot/datetime-inputs.py:                         0.46 sec   0.0 MB
    - ../examples/gallery/plot/multi-parameter-symbols.py:                 0.46 sec   0.0 MB
```

For example, `examples/tutorials/regions.py` takes more than 10 seconds. I guess it's slow because the two global examples plot borders of all countries. However, these borders are not necessary here, as we are showing how to set regions in the tutorial.

I don't check other tutorials, but I believe some tutorials, especially the slowest ones can be much faster if we are more careful about choosing the arguments.

### PR DESCRIPTION
**Pull request by willschlitzer**
Inadvertent creation of new branch due to changing remote git repository. This is the updated version of #868, including @seisman's comments to remove figures.

Fixes #825 

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by seisman** [comment]
We may speed up the 3d perspective tutorial by using a lower-resolution earth relief grid if #873 can be merged.

**Comment by seisman** [comment]
@willschlitzer #873 is already merged. You may try to use low-resolution grids and see if the build time can be reduced.

In the [Cache Data](https://github.com/GenericMappingTools/pygmt/blob/master/.github/workflows/cache_data.yaml) workflow, only `01d`, `30m`, `10m` and `05m` resolution earth relief grids are cached in the CI jobs. So you may try `10m` grid.

File: examples/tutorials/earth-relief.py [hunk 1]
```diff
@@ -21,8 +21,9 @@
 
 ########################################################################################
 # Load sample Earth relief data for the entire globe at a resolution of 30 arc minutes.
```
Comment (seisman) [inline examples/tutorials/earth-relief.py:23]:
```suggestion
# Load sample Earth relief data for the entire globe at a resolution of 1 arc degree.
```

File: examples/tutorials/earth-relief.py [hunk 2]
```diff
@@ -21,8 +21,9 @@
...
-# The other available resolutions are show at :gmt-docs:`datasets/remote-data.html#global-earth-relief-grids`.
-grid = pygmt.datasets.load_earth_relief(resolution="30m")
+# The other available resolutions are show
+# at :gmt-docs:`datasets/remote-data.html#global-earth-relief-grids`.
+grid = pygmt.datasets.load_earth_relief(resolution="01d", registration="gridline")
```
Comment (seisman) [inline examples/tutorials/earth-relief.py:26]: What's the reason to use the gridline-registered grid?
```suggestion
grid = pygmt.datasets.load_earth_relief(resolution="01d")
```

File: examples/tutorials/earth-relief.py [hunk 3]
```diff
@@ -95,8 +96,8 @@
 #
 # In addition to providing global data, the ``region`` argument for
 # :meth:`pygmt.datasets.load_earth_relief` can be used to provide data for a specific
-# area. The ``region`` argument is required for resolutions at 5 arc minutes or higher, and
-# accepts a list (as in the example below) or a string. The geographic ranges are
+# area. The ``region`` argument is required for resolutions at 5 arc minutes or higher,
+# and accepts a list (as in the example below) or a string. The geographic ranges are
 # passed as *xmin*/*xmax*/*ymin*/*ymax*.
 #
 # The example below uses data with a 5 arc minute resolution, and plots it on a
```
Comment (seisman) [inline examples/tutorials/earth-relief.py:103]: For the regional map, could you change the resolution from 05m to 10m?

File: examples/tutorials/earth-relief.py (hunk 2 above)
Comment (willschlitzer) [reply examples/tutorials/earth-relief.py:26]: You had mentioned that the gridline version was the cached file.

File: examples/tutorials/earth-relief.py (hunk 3 above)
Comment (willschlitzer) [reply examples/tutorials/earth-relief.py:103]: Changed in f6f5f54

File: examples/tutorials/earth-relief.py [hunk 4]
```diff
@@ -95,15 +96,15 @@
...
```
Comment (seisman) [inline examples/tutorials/earth-relief.py:103]:
```suggestion
# The example below uses data with a 10 arc minute resolution, and plots it on a
```

File: examples/tutorials/regions.py [hunk 5]
```diff
@@ -96,9 +96,8 @@
 fig.coast(
     region="d",
     projection="Cyl_stere/12c",
-    land="lightgray",
+    land="darkgray",
```
Comment (seisman) [inline examples/tutorials/regions.py:99]: The changes of `regions.py` don't help reduce the time. Could you revert the changes of `regions.py` in this PR? We can explore how to make it faster later.

File: examples/tutorials/3d-perspective-image.py [hunk 6]
```diff
@@ -94,55 +81,23 @@
 
 ########################################################################################
 # The ``perspective`` azimuth can be changed to set the direction that is "up"
-# in the figure.
+# in the figure. The ``contourpen`` parameter sets the pen used to draw contour lines
+# on the surface. :meth:`pygmt.Figure.colorbar` can be used to add a color bar to the
+# figure. The ``cmap`` argument does not need to be passed again. To keep the color
+# bar's alignment similar to the figure, use ``True`` as the ``perspective`` argument.
```
Comment (seisman) [inline examples/tutorials/3d-perspective-image.py:87]: See #886 for discussions and please replace "argument" with "parameter" in other instances of these two tutorials.
```suggestion
# figure. The ``cmap`` parameter does not need to be passed again. To keep the color
# bar's alignment similar to the figure, use ``True`` as the ``perspective`` parameter.
```

File: examples/tutorials/regions.py (hunk 5 above)
Comment (willschlitzer) [reply examples/tutorials/regions.py:99]: Updated in dfc3dee; I re-added the borders argument

**Review by seisman — APPROVED** [review]

**Pull request merged**